console/src/components/Services/Data/TableModify/utils.js

```diff
@@ -82,4 +82,15 @@ export const sanitiseRootFields = rootFields => {
   return santisedRootFields;
 };
 
+export const sanitiseColumnNames = columnNames => {
+  const sanitised = {};
+  Object.keys(columnNames).forEach(c => {
+    const trimmedCustomName = columnNames[c] ? columnNames[c].trim() : null;
+    if (trimmedCustomName) {
+      sanitised[c] = columnNames[c].trim();
```

Comment:
`trimmedCustomName` is either `columnNames[c].trim()` or `null`. So inside of the `if` statement it'll be the the first one. Why doing `trim` again?